streaming-binlog/src/test/java/org/openmrs/analytics/FhirConverterTest.java

```diff
@@ -14,7 +14,12 @@
 
 package org.openmrs.analytics;
 
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.times;
```

Comment:
Please be consistent; either don't use static imports for mockito or if you do then replace all `Mockito.` references in this file with static imports.